ui/ops/src/routes/ops/waste/WasteTable.vue

```diff
@@ -16,7 +16,7 @@
           {{ timestampToDate(item.wasteCreateTime).toLocaleString() }}
         </template>
         <template #item.weight="{ item }">
-          {{ item.weight.toFixed(2) }} {{ uiConfig.config.wasteRecordUnit ?? "kg" }}
+          {{ item.weight.toFixed(2) }} {{ uiConfig.config.ops.waste.unit ?? "kg" }}
```

Comment:
you probably want to use `?.` here otherwise an absent ops or waste object will cause null pointer exceptions:
`uiConfig.config?.ops?.waset?.unit ?? 'kg'`.

Same further down the file